src/components/fieldset/fieldset.yaml

```diff
@@ -1,3 +1,42 @@
+params:
+- name: describedBy
+  type: string
+  required: false
+  description: Text or element id to add to the `aria-describedby` attribute to provide description of the group of fields for screenreader users.
+- name: legend
+  type: object
+  required: false
+  description: Options for the legend
+  params:
+  - name: text
+    type: string
+    required: true
+    description: If `html` is set, this is not required. Text to use within the legend. If `html` is provided, the `text` argument will be ignored.
+  - name: html
+    type: string
+    required: true
+    description: If `text` is set, this is not required. HTML to use within the legend. If `html` is provided, the `text` argument will be ignored.
+  - name: classes
+    type: string
+    required: false
+    description: Classes to add to the legend container.
```

Comment:
```diff
- container
```
?

Reply:
Sorry, realise this is the same in the README – happy for you to leave these if you prefer.